docs/source/conf.py

```diff
@@ -53,6 +53,7 @@
 myst_heading_anchors = 2
 
 intersphinx_mapping = {
+    "kedro": ("https://docs.kedro.org/en/stable/", None),
```

Comment:
We might want to fix a version here, to avoid undesirable build errors

```suggestion
    "kedro": ("https://docs.kedro.org/en/0.18.14/", None),
```

Reply:
I'm not sure about this @astrojuanlu -- I know what you mean but hear me out. If we pin to a version, and it doesn't exist, then the links fail, right? So if we want to show the 0.19.0 version, we have to wait til it's out, then update here, then **re-release** these viz docs to point to 0.19.0. This is a kind of coupling. It allows Viz to release when they want but still limits it to a schedule alongside the framework to reflect latest and greatest framework docs.

Would it not be more suitable to point to stable and just ensure redirects manage any updated page issues? I truly don't know any more 🤷

Reply:
> If we pin to a version, and it doesn't exist, then the links fail, right?

Yes

> So if we want to reference pages in the 0.19.0 framework docs, we have to wait til it's out, then update here, then re-release these viz docs to point to 0.19.0, right?

This I don't understand: if we want to point to a 0.19 page that _will_ exist but doesn't yet, the link will be broken anyway right?

On the other hand, if we point to a page that doesn't move, using `stable` or a specific version doesn't matter.

But yes, this means that when we want to update the docs, there is some coupling. Maybe it's better to leave it this way for now.

Reply:
> This I don't understand: if we want to point to a 0.19 page that will exist but doesn't yet, the link will be broken anyway right?

Reply:
> This I don't understand: if we want to point to a 0.19 page that will exist but doesn't yet, the link will be broken anyway right?

Yes, ofc. But if we point to a page that does exist in 0.18.14 but it resolves to a 0.19.0 link, it won't work

https://docs.kedro.org/en/0.18.14/resources/index.html is fine but https://docs.kedro.org/en/0.19.0/resources/index.html resolves to the not found page

> On the other hand, if we point to a page that doesn't move, using stable or a specific version doesn't matter.
So I think it **does** matter, unless there's cleverness inside RTD where it takes the pinned version and resolves down to stable??

Reply:
So to get Viz to point to 0.19 docs we'd have to release it with a new pinned version after framework 0.19 goes out. If we point to stable, we don't. So let's leave it pointing to stable and see what terrors this unleashes instead.